stable/datadog/README.md

```diff
@@ -45,7 +45,7 @@ The following table lists the configurable parameters of the Datadog chart and t
 | `image.tag`                 | The image tag to pull              | `6.3.2`                                   |
 | `image.pullPolicy`          | Image pull policy                  | `IfNotPresent`                            |
 | `rbac.create`               | If true, create & use RBAC resources | `true`                                  |
-| `rbac.serviceAccount`       | existing ServiceAccount to use (ignored if rbac.create=true) | `default`       |
+| `rbac.serviceAccountName`   | existing ServiceAccount to use (ignored if rbac.create=true) | `default`       |
```

Comment:
@timfallmk 
Please sign off the commits. Also the chart needs a version bump.
But I also noticed there is a glitch in the chart, where the serviceAccount is created with fullname, but the daemonset/deployments use either fullname or the given name - could you fix that too? See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md

Reply:
@davidkarlsen I'm not sure I understand the need for a text sig (as opposed to a crypto sig), but I'll add one.

Could you point to where you see the mismatch in names? It looks from the `deamonset` and `deployment` charts that if `values.serviceAccountName` is set, that is used, otherwise the default `datadog.fullname` is used. Were you looking for a different selection method?

Reply:
What I suggest is that everywhere where the service account is referenced the same template function from the helper should be used.

Reply:
See https://www.helm.sh/blog/helm-dco/index.html for instructions on the DCO

Reply:
I see what you're getting at, not necessarily a bug, but a duplication of function. I'll take a look.

Reply:
did you have a chance to take a look. I think it's just one place it does not refer the template

Reply:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - where a template function "mychart.serviceAccountName" is used to generate the serviceAccountName consistently